base/uk.ac.stfc.isis.ibex.ui.configserver/src/uk/ac/stfc/isis/ibex/ui/configserver/editing/iocs/dialog/IocDialog.java

```diff
@@ -114,87 +64,47 @@ private void nextPage() {
      *            The configuration currently being edited.
      * @param ioc
      *            The IOC to add.
-     * @param isNew
-     *            Flag indicating whether IOC to edit is newly added or
-     *            pre-existing.
+     * @param readOnly
+     *            Specifies whether the settings are editable
      */
-    public IocDialog(Shell parent, EditableConfiguration config, EditableIoc ioc, boolean isNew) {
+    public IocDialog(Shell parent, EditableConfiguration config, EditableIoc ioc, boolean readOnly) {
```

Comment:
Can we use ioc.isEditable() instead? I think that means we could remove readOnly from here and in IocOverviewPanel